### ISSUE TITLE
Full text search should include notes with the searched text in a tag

### ISSUE THREAD

**Issue by mlncn** [standardnotes/forum#567]
That is, if i have a note titled "A People's History of the United States" with content "Excellent book; kept in our social justice library" and tagged "USA"— when i search for "USA" i expect this note to come up even though "USA" is not in the title nor the content, just the tag.

(Not sure if #64 was ever interpreted this way, but in any case this does not currently work on the hosted version at standardnotes.org.)

**Comment by dev-matteo** [comment]
I agree with all this opinions. Tags are made to access the information and not to organize like a folder. If you have a folder with notes, maybe a tag with the same value could be correct. If i want to create a note in a sub-folder, i probably want the note to be tagged with the relative path of the folders like a tree. When i search i would like to find all the notes that have the specific tag or maybe contains a part of it.  Example:   **programming.css** and **code.css** 
If i search for **css** i want to find both ne note. This is what i expect.

**Comment by dig12345** [comment]
Would really like to see this implemented, either as part of the main search box or a separate search box. I have lots of nested folders and its a real pain to find the correct folder when looking for notes without search.
